app/cni/cmd/install/main.go

```diff
@@ -1,241 +1,9 @@
 package main
 
 import (
-	"bytes"
-	"io/ioutil"
-	"os"
-	"os/signal"
-	"path"
-	"path/filepath"
-	"syscall"
-	"time"
-
-	"github.com/natefinch/atomic"
-	"github.com/pkg/errors"
-	"go.uber.org/multierr"
-
-	"github.com/kumahq/kuma/pkg/core"
-	"github.com/kumahq/kuma/pkg/util/files"
-)
-
-const (
-	kumaCniBinaryPath  = "/opt/cni/bin/kuma-cni"
-	primaryBinDir      = "/host/opt/cni/bin"
-	secondaryBinDir    = "/host/secondary-bin-dir"
-	serviceAccountPath = "/var/run/secrets/kubernetes.io/serviceaccount"
-)
-
-var (
-	log = core.NewLoggerWithRotation(2, "/tmp/install-cni.log", 100, 0, 0).WithName("install-cni")
+	"github.com/kumahq/kuma/app/cni/pkg/install"
```

Comment:
Note for reviewers: this changed into a simple call, not sure why git detected it as change, this should be a new file + a move.